src/main/java/seedu/address/logic/Messages.java

```diff
@@ -15,6 +15,7 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSONS_DISPLAYED_INDEX = "The person indexes provided are invalid";
```

Comment:
Good to show the invalid index.

Reply:
Okay, I will add it as a patch in a different PR